src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts

```diff
@@ -18,7 +20,7 @@ export function handleCopyRequest(clipboardService: IClipboardService, range: Sl
 			}
 
 			if (i !== range.toRow) {
-				results += os.EOL;
+				results += textResourcePropertiesService.getEOL(URI.from({ scheme: Schemas.untitled, path: '1' }));
```

Comment:
>path: '1' [](start = 89, length = 9)

small nit - but my personal preference would be to leave this undefined instead of passing in 1 (assuming that still behaves the same). Passing in a value is just going to make people wonder if that has any special significance which isn't clear from its usage here

Reply:
I copied this from how they do this copying in the debug output; presumable leaving path empty would work but I would have to test.